extensions/mssql/src/main.ts

```diff
@@ -121,6 +123,8 @@ export async function activate(context: vscode.ExtensionContext): Promise<MssqlE
 	const bookContributionProvider = getBookExtensionContributions(context);
```

Comment:
You can remove this now right?

Reply:
It needs to be put somewhere. This is the actual implementation of finding books.